backend/src/test/java/gov/cdc/usds/simplereport/api/PatientManagementTest.java

```diff
@@ -7,21 +7,44 @@
 import java.io.IOException;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import gov.cdc.usds.simplereport.db.model.Facility;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.service.OrganizationService;
+import gov.cdc.usds.simplereport.test_util.TestDataFactory;
+
+
 /**
  * Tests for adding and fetching patients through the API
  */
 public class PatientManagementTest extends BaseApiTest {
 
+    @Autowired
+    private TestDataFactory _dataFactory;
+    @Autowired
+    private OrganizationService _orgService;
+
     @BeforeEach
     public void setup() {
         truncateDb();
     }
 
+    @Test
+    public void queryPatientWithFacility() throws Exception {
+        Organization org = _orgService.getCurrentOrganization();
+        Facility place = _orgService.getFacilities(org).get(0);
+        _dataFactory.createMinimalPerson(org, place, "Cassandra", null, "Thom", null);
+        _dataFactory.createMinimalPerson(org, null, " Miriana", "Linas", "Luisito", null);
+        JsonNode patients = fetchPatientsWithFacility();
+        assertEquals(true, patients.get(0).get("facility").isNull());
```

Comment:
This can also just be `assertTrue`, but that method literally just delegates to this one. 

There's also a `path` method that lets you traverse possibly empty paths without getting exceptions until you check, but I don't think you need it here, that's just FYI. 🙂